lib/models/timetable/course_model.dart

```diff
@@ -30,6 +37,9 @@ class CourseModel implements Comparable<CourseModel> {
     venue = json['venue'];
     midsem = json['midsem'];
     endsem = json['endsem'];
+    midSemVenue = json['midSemVenue'];
+    endSemVenue = json['endSemVenue'];
+    timings = json['timings'];
```

Comment:
You cannot map json['timimgs'] directly you need to appropriately convert it to a map<string,dynamic> and then map it

Reply:
fixed